### ISSUE TITLE
Embeddings

### ISSUE THREAD

**Issue by moss-co** [Walter0807/MotionBERT#78]
Hi, 
I have been trying to use train_action_1shot.py to get those using this [model config](https://github.com/Walter0807/MotionBERT/blob/main/configs/action/MB_train_NTU120_oneshot.yaml), but challenged by the machine config. 

Is there any any other approach you recommend we get the embeddings  ? I also see that the checkpoints for mainly the model params. How can I get the embeddings from these checkpoints ? 

Also, perhaps do you have the pre-trained embeddings available?

**Comment by Walter0807** [comment]
Hi, do you want to obtain the pretrained motion representations or do you want to fine-tune MB for one-shot action recognition?

**Comment by moonshot-s** [comment]
hey - was Iooking for pre-trained motion representations. Also in ActionNet, input dim ( Input: (N, M, T, J, C)) what does M represent ?

**Comment by Walter0807** [comment]
> hey - was Iooking for pre-trained motion representations.

https://github.com/Walter0807/MotionBERT#using-motionbert-for-human-centric-video-representations 
This section should be what you are looking for.

**Comment by Walter0807** [comment]
> Also in ActionNet, input dim ( Input: (N, M, T, J, C)) what does M represent ?

It's the dimension of person numbers.

**Comment by moss-co** [comment]
thank you for clarifying! appreciate it. 
Should we expect the network to learn similar embeddings/feature representations when training both ActionNet and MeshRegressor from just the backbone without the head ? 

I think I am also not clear on what xsubject and xview mean here (giving M = 2) , I assumed we are working with single human per input pose representations ?
